service-13/PUS-13-16.acn

```diff
@@ -0,0 +1,34 @@
+--!
+--! Copyright (C) 2018 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-13-16 DEFINITIONS ::= BEGIN
+
+TM-13-16-LargePacketUplinkAbortionReport []
+{
+    largeMessageTransactionIdentifier [],
+    largePacketFailureReason []
+}
+
+LargePacketFailureReason [size 32, encoding pos-int, endianness big, encode-values]
```

Comment:
why size 32? (if it's a common practise in lib - report issue and we have to fix that, 32 bit enums seems like overkill :) )

Reply:
At least some enums are of 32 bits size, so I will create task to change it.